op-node/rollup/derive/engine_queue.go

```diff
@@ -413,6 +426,7 @@ func (eq *EngineQueue) logSyncProgress(reason string) {
 		"l2_safe", eq.ec.SafeL2Head(),
 		"l2_safe_pending", eq.ec.PendingSafeL2Head(),
 		"l2_unsafe", eq.ec.UnsafeL2Head(),
+		"l2_backupUnsafe", eq.ec.BackupUnsafeL2Head(),
```

Comment:
nit: everywhere else is snake case, this mixes camel case and snake case

Reply:
Fixed at https://github.com/ethereum-optimism/optimism/pull/8925/commits/9709ec0e25504f6628d4350a7674ab7e4c83b281.

Reply:
Fixed at https://github.com/ethereum-optimism/optimism/pull/8925/commits/bd08911843ade9a1f0b9f864413551ba4b4226b2.